docs/source/sponsors/discounts/discounts.rst

```diff
@@ -0,0 +1,135 @@
+FTC Team Discounts
+==================
+
+As a benefit of being a FIRST Tech Challenge Team, some vendors and sponsors of
+*FIRST* programs have generously provided discounts on products to teams.
+These discounts may be for specific categories of products, or may apply
+site-wide, and may require an application process or a season-specific or
+team-specific promo code. Read details below for each vendor for more
+information.
+
+.. dropdown:: FTC Team Grant Opportunities
+
+   .. image:: images/ftc-logo.png
+      :height: 50px
+      :target: https://www.firstinspires.org/robotics/team-grants
+      :alt: FTC Onecolor Logo
+      :name: FTC Onecolor Logo
+      :class: invert-img
+
+   |
```

Comment:
Add another line break and in all other similar places

Reply:
I DID. After each logo, I added a linebreak to create the necessary spacing.

Reply:
I _did_